src/pages/Shop/Components/ShopFilter/ShopFilter.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+
+import './ShopFilter.scss'
+class ShopFilter extends Component {
+  render() {
```

Comment:
```suggestion
import React, {Component} from 'react';
import './ShopFilter.scss'

class ShopFilter extends Component {
  render() {
```
귀찮다고 생각될 수 있겠지만 모든 코드 작성의 시작은 컨벤션입니다. 앞선 리뷰들을 보니 반복적으로 안지켜지고 있는 부분 같습니다. 상호간에 읽기 쉽고, 소통이 쉬운 코드를 작성할 수 있도록 배려해주세요.

Reply:
넵 수정하였습니다!